test/controllers/PaymentScheduleControllerSpec.scala

```diff
@@ -264,7 +265,7 @@ class PaymentScheduleControllerSpec extends ItSpec with PegaRecreateSessionAsser
                   s"""
                  |{
                  |        "correlationId": "8d89a98b-0b26-4ab2-8114-f7c7c81c3059",
-                 |        "origin": "Bta",
+                 |        "origin": "${origin.toString().takeRight(3)}",
```

Comment:
this logic could be adapted to look more like the `AuditService` logic like you did in another test